src/actuator/ActuatorFAST.C

```diff
@@ -682,6 +735,361 @@ ActuatorFAST::execute()
   stk::mesh::parallel_sum_including_ghosts(bulkData, sumFieldG);
 }
 
+
+// Copmute the filtered lifting line theory correction
+void ActuatorFAST::filtered_lifting_line()
+{
+
+
+  // The number of dimensions (assumes 3D)
+  int nDim=3;
+
+  // The total number of turbines
+  const size_t numTurbines = fi.nTurbinesGlob;
+
+  // The index of actuator point
+  // This identifiex an actuator point in a vector containing all actuator
+  // points in the simulation
+  size_t np;
+
+
+  // Loop through all turbines
+  for (size_t iTurb=0; iTurb < numTurbines; iTurb++ )
+  {
+
+    // Only use if correction is active for this turbine
+    // Point to the last element of this array that was just appended
+    auto actuatorFASTInfo = dynamic_cast<ActuatorFASTInfo*>(
+      actuatorInfo_.at(iTurb).get());
+
+    // If the correciton is not active for this turbine, skip this turbine
+    if ( ! actuatorFASTInfo -> fllt_correction_) continue;
+
+    // Do not consider this unless the turbine lies in the same processor
+    if (FAST.get_procNo(iTurb) != NaluEnv::self().parallel_rank()) continue;
+
+    // Number of blades
+    const size_t numBlades = FAST.get_numBlades(iTurb);
+
+    // The total number of actuator points per blade
+    const size_t ptsPerBlade = FAST.get_numForcePtsBlade(iTurb); //totalActuatorNodes / numBlades;
+
+    ///////////////////////////////
+    // Step 1: Compute function G
+    ///////////////////////////////
+
+    // The force at the given actuator point
+    std::vector<double> force(nDim);
+    // The velocity at a given actuator point
+    std::vector<double> vel(nDim);
+
+    // Loop through all blade points
+    for (size_t nb=0; nb < numBlades; nb++)
+      {
+      // Loop through all blade points
+      for (size_t na=0; na < ptsPerBlade; na++)
+        {
+
+          // The actuator point index
+          np = indexMap_[iTurb][nb][na];
+
+          // actuator line info object of interest
+          auto infoObject =
+            dynamic_cast<ActuatorFASTPointInfo*>(
+              actuatorPointInfoMap_.at(np).get());
+        
+          // Get the force from FAST
+          FAST.getForce(force, np, infoObject->globTurbId_);
+          // Get the velocity from FAST
+          FAST.getRelativeVelForceNode(vel, np, infoObject->globTurbId_);
+
+          // Zero the radial component of the velocity and force
```

Comment:
Not critical, but removing some of these stale comments from debugging would be nice.